src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeDateFormatter.java

```diff
@@ -20,18 +20,43 @@ public static String getCurrentDateTime() {
             DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"));
     }
 
-    public static String getIacDecisionDate(String hoDecisionDate) {
+    public static String getIacDateTime(String homeOfficeDate) {
         try {
-            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
-            LocalDate parsedDate = LocalDate.parse(hoDecisionDate, formatter);
-            formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
-            return parsedDate.format(formatter);
+            if (homeOfficeDate != null) {
+                //only Date is provided
+                DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
This can be static final. I think it is relatively an expensive operation and better keep it at class level.

Reply:
Changed as suggested